packages/postgres-database/src/simcore_postgres_database/models/products_prices.py

```diff
@@ -0,0 +1,45 @@
+import sqlalchemy as sa
+
+from ._common import NUMERIC_KWARGS, column_created_datetime
+from .base import metadata
+from .products import products
+
+#
+# - Every product has an authorized price
+# - The price is valid from the creation date until a new price is created
+# - No rows are deleted!
```

Comment:
minor: you can enforce that using a constraint or a trigger that always returns false when you try to delete....

Reply:
this is now constraint by products itself. Nonetheless, I would like to know more about the technique you mention